src/coreclr/src/libraries-native/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+set(CORECLR_BUILD ON)
+
+if(CLR_CMAKE_HOST_UNIX AND NOT DEFINED CLR_CROSS_COMPONENTS_BUILD)
+    add_subdirectory(../../../libraries/Native/Unix/System.Globalization.Native System.Globalization.Native)
```

Comment:
>../../../l [](start = 21, length = 10)

We don't have the repo root path stored in any variable instead of using relative path like that?

Reply:
Nope, but I'm going to add it.